YouVsVirus/Assets/Scripts/levelgethome/CreatePopLevelgethome.cs

```diff
@@ -94,12 +94,8 @@ private void CreateHumans()
             // place player near top left edge of the screen (using the not shrunk screen bounds)
             Vector2 coords;
             
-            // Create random coordinates and check whether they are spawnable
-            do
-            {
-                coords = new Vector2(-randomGridForHumans.screenBounds.x / 0.8f + 0.2f, randomGridForHumans.screenBounds.y / 0.8f - 0.2f);
-            }
-            while (!nonSpawnableSpaceClass.coordinatesAreSpawnable2D(coords));
+            // Create human in upper left corner
+            coords = new Vector2(-randomGridForHumans.screenBounds.x / 0.8f + 0.2f, randomGridForHumans.screenBounds.y / 0.8f - 0.2f);
 
             Vector3 coords3D = new Vector3 (coords[0], coords[1], 0);
```

Comment:
Why copy this into another vector?

Reply:
Good question.
We use the 3D instantiate version here which needs a 3D vector.
I dont know why and i dont want to change it.
However, our nonSpawnableSpace accepts a 2D vector as input (since its logical, since we work in 2D).
So i had to change the initial 3D vector to a 2D vector to pass it to the nonSpawnableSpace, but bc i didnt want to touch the Instantiate call i needed to convert it back into 3D before call Instantiate.

Reply:
unity is weird, since it sometimes allows for 2d vector and sometimes not